### PR TITLE
Add help text to mount HDFS dialog

### PR DESCRIPTION
**Pull request by kevcunnane**
Update help text for Mount dialog. As this relies on the help text fix to be in, comparing against that branch for now and will refresh with compare against master once that's committed in.

### CODE REVIEW

File: extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts [hunk 1]
```diff
@@ -390,15 +390,24 @@ export class MountHdfsDialog extends HdfsDialogBase<MountHdfsProperties> {
...
+						info: localize('mount.hdfsPath.info', "Path to a new (non-existing) directory which you want to associate with the mount")
+					}
 				}, {
 					component: this.remoteUriInputBox,
-					title: localize('mount.remoteUri', "Remote URI"),
-					required: true
+					title: localize('mount.remoteUri.title', "Remote URI"),
+					required: true,
+					layout: {
+						info: localize('mount.remoteUri.info', "The URI to the remote data source. Example for ADLS: abfs://fs@saccount.dfs.core.windows.net/")
```
Comment (chlafreniere) [inline extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: Should we point to a docs page instead? Is ADLS included because it's going to be the typical use case here? Also, any reason for "saccount" instead of "account"?

File: extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts [hunk 2]
```diff
@@ -390,15 +390,24 @@ export class MountHdfsDialog extends HdfsDialogBase<MountHdfsProperties> {
...
+					}
 				}, {
 					component: this.credentialsInputBox,
-					title: localize('mount.credentials', "Credentials"),
-					required: false
+					title: localize('mount.credentials.title', "Credentials"),
+					required: false,
+					layout: {
+						info: localize('mount.credentials.info', "Mount credentials for authentication to remote data source, for access to read the remote data")
```
Comment (chlafreniere) [inline extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: "for access to read" seems a bit wordy to me. trying to figure out why.

Comment (kevcunnane) [reply extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: I'll ask Nellie to review these 😄

File: extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts (hunk 1 above)
Comment (NelGson) [reply extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: Yes, ADLS is a common option. You can remove the specific example if you want. Let's just do that.

File: extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts (hunk 2 above)
Comment (NelGson) [reply extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: Yes, can be shortened to: Mount credentials for authentication to remote data source for reads.

File: extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts (hunk 1 above)
Comment (kevcunnane) [reply extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: I actually like it, is it OK to leave for now? It's helpful to know what to look for.

File: extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts (hunk 2 above)
Comment (kevcunnane) [reply extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts]: Done

**Comment by kevcunnane** [comment]
I'm closing this and will reopen a new PR against master directly, since the branch was struggling to handle new changes.
